docs/source/blueprints/BP002.rst

```diff
@@ -8,7 +8,7 @@ BP002
 :Reviewers: Beraldo Leal, Cleber Rosa
 :Created: 27-Jan-2020
 :Type: Architecture Blueprint
-:Status: Approved
+:Status: WIP
```

Comment:
OK, this matches BP000 definitions.  Unfortunately, we have not the same for the other BPs.  @beraldoleal is there an exception for Epic blueprints or should BP001 also receive such an update?

BTW, just to make it clear, I'm not suggesting to do it here on this PR.

Reply:
We need to update, there is no exception.